CCG salt jinja templating #16905 keichwa (#2420)
uyuni-project/uyuni-docs: modules/client-configuration/pages/configuration-management.adoc

You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.

[NOTE]
====

> Some of the following features are available only for traditional clients.

Reviewer: Hmm, this is for "master" and there we drop traditional stack.

Author: Yes, there are still many such references in master.  This file requires a general walkthru.  I'd like to backport (4.3) it first, and then prune this file (chapter).

Reviewer: makes sense

== Compare Configuration Files

You can also use the system set manager (SSM) to compare the configuration file deployed on clients with the configuration file stored on the {productname} Server.

.Procedure: Comparing Configuration Files
. In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the clients subscribed to the configuration files you want to compare.
. Navigate to menu:Systems[System Set Manager], and go to the menu:Configuration[Compare Files] subtab to the list of available configuration files.
. OPTIONAL: Click the number in the [guimenu]``Systems`` column to see which clients are currently subscribed to the configuration file.
. Check the configuration files to compare, and click btn:[Schedule File Comparison].

== Jinja templating on {salt} Clients

Jinja templating is possible on {salt} Clients.

> Jinja variables can be used in {salt} pillars or grains in a configuration file of a {salt} state channel.

Reviewer: I do not understand this sentence. This does not sound correct.

Do you mean something like this?

Jinja provide variables (made) from pillars or grains. They can be used in configuration files or salt states.

> Jinja variables can be used in {salt} pillars or grains in a configuration file of a {salt} state channel.

Author: ```suggestion
Jinja provides variables from pillars or grains.
They can be used in configuration files or {salt} states.
```
Thanks for clarification!
